index.js

```diff
@@ -23,8 +28,7 @@ module.exports = function (input, length) {
 
 	while (i--) {
 		var x = pathParts[i];
-
-		if (remainingLength < x.length) {
+		if (remainingLength < x.length + 1) {
```

Comment:
[This PR](https://github.com/sindresorhus/truncate-url/pull/1/files#diff-168726dbe96b3ce427e7fedce31bb0bcL29) introduced this bug